src/components/PageSettingsToggle.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div data-width="1/3" class="k-column">
+    <div class="k-toggle-field k-field">
+      <div data-theme="field" data-type="toggle" class="k-input">
+        <span class="k-input-element">
+          <label :data-disabled="setting.disabled" class="k-toggle-input">
+            <input
+              ref="input"
+              :checked="setting.value"
+              :id="setting.id"
+              :disabled="setting.disabled"
+              class="k-toggle-input-native"
+              type="checkbox"
+              @change="change($event.target.checked)"
+            >
+            <span class="k-toggle-input-label" v-html="label" />
+          </label>
+        </span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+
+  props: {
+    setting: Object,
+  },
+
+  computed: {
+    label() {
+      if (Array.isArray(this.setting.text)) {
+        return this.setting.value ? this.setting.text[0] : this.setting.text[1];
+      }
+      return this.setting.text;
+    },
+  },
+
+  methods: {
+    change(checked) {
+      this.setting.value = checked;
+      this.$emit('change', this.setting.id, checked);
+    }
+  }
+
+}
+</script>
+
+<style lang="scss">
+.k-commentions-pagesettings .k-toggle-input .k-toggle-input-label {
+  font-size: .875rem;
+}
+.k-commentions-pagesettings .k-toggle-input[data-disabled="true"] .k-toggle-input-label {
+  color: #ccc;
+}
+.k-commentions-pagesettings .k-toggle-input-native:not([disabled]) {
```

Comment:
I’m a bit undecided, whether I like the colored toggle buttons. They don’t look very Kirby-like, but are in fact very helpful! :-) Maybe we could move them to a settings-sidebar or the header in a futute release. Let’s keep them this way for now. I also thought about mass-editing already, but this is a whole different story, that we should not tackle by now.

Reply:
I hear you - this was an exploration, and I felt the same. I removed the colours for now - can be revisited at a later stage.